### PR TITLE
Add noise fine tuning

### PR DESCRIPTION
**Pull request by LukasNickel**
Add config keys that trigger the use of `lstchain_tune_nsb` in order to get the image modifier settings for a given observation run.
That should eliminate one step got getting matching monte carlo

### CODE REVIEW

**Comment by LukasNickel** [comment]
Connected issue: #160

**Comment by garciagenrique** [comment]
Thanks for the PR @LukasNickel, looks fine to me, just a silly suggestion.

File: lstmcpipe/config_MC_prod.yml [hunk 1]
```diff
@@ -27,6 +27,11 @@ obs_date: 20200629_prod5_trans_80
   # pro3 : `20190415`
 zenith: zenith_20deg  # Taken into account only when `prod_type: prod5` [ thus `obs_date` : 20200629_prod5(_trans_80) ]
 pointing: south_pointing
+
+# If the image modifier settings should be calculated on the fly
```
Comment (garciagenrique) [inline lstmcpipe/config_MC_prod.yml:31]: The image modifier settings will only work when running a `dl1ab` stage, so I would put `dl1_reference_id`, `dl1_noise_tune_data_run`, `dl1_noise_tune_mc_run` close in the config file.

**Comment by garciagenrique** [comment]
Now that I'm thinking it twice, what would you say about creating 2 `config_MC_prod.yml` files ?
 - One shows how to trigger a "normal" MC prod (starting from `r0_dl1`) - and named `config_MC_prod.yml`
 - The second one shows how to trigger a MC prod starting from dl1ab stage, show (commented) examples of all dl1ab options, i,.e., `dl1_reference_id`, 'dl1_noise_tune_data_run' and `dl1_noise_tune_mc_run`, as well as the `dl1ab` added into the `stages_to_be_run`

**Comment by LukasNickel** [comment]
> 

Good idea. Maybe we should start separating both things

**Comment by garciagenrique** [comment]
> Good idea. Maybe we should start separating both things

Could you then take care of this in this PR, @LukasNickel ? (no rush :) ! )

File: lstmcpipe/config_MC_prod.yml [hunk 2]
```diff
@@ -18,33 +18,23 @@ workflow_kind: lstchain  # Not expected to be changed by the user
 base_path: /fefs/aswg/workspace/MC_data_simlink  # Default user path
   # lstanalyzer user : /fefs/aswg/data/mc
   #
-# Only for dl1 reprocessing
-#dl1_reference_id: #old id including versions (full directory name)
 prod_type: prod5
-  # Choose between {prod5, prod3}
 obs_date: 20200629_prod5_trans_80
-  # prod5 : `20200629_prod5` or `20200629_prod5_trans_80`
```
Comment (garciagenrique) [inline lstmcpipe/config_MC_prod.yml:26]: Sorry for being picky here. 
I would suggest to keep some (a less as possible) comments indicating the possible values of certain fields  (users usually are not aware of how many MC simtel prods are). 

Maybe `prod_type` and `obs_date` ? What do you think ? Not sure if anybody would like to run old prods, indeed...

Comment (LukasNickel) [reply lstmcpipe/config_MC_prod.yml:26]: But why would anyone run anything else than these values?
Its not like we have 200 simulations currently

Comment (LukasNickel) [reply lstmcpipe/config_MC_prod.yml:26]: But maybe obs date is not self-explanatory indeed

Comment (LukasNickel) [reply lstmcpipe/config_MC_prod.yml:26]: If they dont know the productions they should stick to the defaults though I think

Comment (garciagenrique) [reply lstmcpipe/config_MC_prod.yml:26]: Fair enough ! you convinced me :)

**Review by garciagenrique — APPROVED** [review]

**Pull request merged**